k8s/kong_plugins/http_mirror/schema.lua

```diff
@@ -2,25 +2,20 @@ local typedefs = require "kong.db.schema.typedefs"
 
 return {
     name = "http-mirror",
-    fields = {
-        {
-            consumer = typedefs.no_consumer
-        },
-        {
-            config = {
-                type = "record",
-                fields = {
-                    {
-                        mirror_to = {
-                            type = "array",
-                            required = true,
-                            elements = {
-                                type = "string"
-                            }
-                        }
+    fields = {{
```

Comment:
What this PR have to be doing with mirroring feature?

Reply:
The mirroring feature is also a Kong plugin, so I took the liberty to make both Kong plugins we have look consistent with each other